rasa/cli/test.py

```diff
@@ -115,29 +116,51 @@ def run_core_test(args: argparse.Namespace) -> None:
     )
 
 
-def run_nlu_test(args: argparse.Namespace) -> None:
-    """Run NLU tests."""
+async def run_nlu_test_async(
+    config: Optional[Union[Text, Dict]],
+    data_path: Optional[Text],
+    models_path: Optional[Text],
+    output_dir: Optional[Text],
+    cross_validation: Optional[bool],
+    percentages: Optional[List[int]],
```

Comment:
As `compare_nlu_models` only accepts a `List[int]` this should also only be able to accept `List[int]`
```suggestion
    percentages:List[int],
```

Reply:
Changed it!